### DISCUSSION TITLE
HSS payload output file

### DISCUSSION THREAD

**Discussion by Shivaraj199** [polarfire-soc/polarfire-soc-documentation#299] [HSS and U-Boot] [answered]
Hello,
I am trying to generate to HSS payload , but i am not able to locate the output.bin file in the directoey I build

LOG:
   
     /hart-software-services/tools/hss-payload-generator/test$ ./hss-payload-generator -c config.yaml output.bin  
     Hart Software Service formatted boot image generator v0.99.35
     Copyright (c) 2021-2022 Microchip FPGA Embedded Systems Solutions.
     yaml_parser_parse(): Success

it says success but I dont see any genrated file?

**Comment by hughbreslin** [comment, marked as answer]
Hey @Shivaraj199 

the 'yaml_parser_parse(): Success' probably just means its parsed your yaml file, it doesn't mean the tool has completed successfully. This is an output from a sample run I just did:

```bash
root@IRD-LT-M31864B:/mnt/c/work/payload/hss-payload-generator/hss-payload-generator/binaries# ./hss-payload-generator -c config.yaml output.bin
Hart Software Service formatted boot image generator v0.99.35
Copyright (c) 2021-2022 Microchip FPGA Embedded Systems Solutions.

NOTICE: mpfs-hal-ddr-demo.elf: ignoring >>exec-addr=0xb0000000<< as payload is an ELF file

Set-name is >>PolarFire-SoC-HSS::TestImage<<
Output filename is >>output.bin<<
root@IRD-LT-M31864B:/mnt/c/work/payload/hss-payload-generator/hss-payload-generator/binaries# ls
LICENSE.md  SHA512SUM    hss-payload-generator      mpfs-hal-ddr-demo.elf  output.bin
README.md   config.yaml  hss-payload-generator.exe  msys-2.0.dll
root@IRD-LT-M31864B:/mnt/c/work/payload/hss-payload-generator/hss-payload-generator/binaries#
```

I would suggest double checking paths etc in your yaml and see if that could be the issue. In my case I have all of the files in the same directory, its not neat but handy to test with.

**Reply by Shivaraj199** [reply]
Yes keeping the same directory did work.
I was trying to load u-boot.bin instead of uboot

**Reply by hughbreslin** [reply]
Its always the little things... Glad its sorted :)
